coops-ui/src/mixins/common-mixin.ts

```diff
@@ -12,7 +12,7 @@ export default class CommonMixin extends Vue {
    * @param baseObj The base object.
    * @param prop The property to be removed.
    */
-  omitProp (baseObj: object, prop: Array<string>): Object {
+  omitProp (baseObj: object, prop: Array<string>): object {
```

Comment:
Good catch, i must have missed this when we discussed this before, thank you!

Reply:
Yeah, I also learned that I should use `number` instead of `Number`.

The basic rule, I believe, is to use the lower case name for type declarations, but the upper case name when using `new Number()` or when using any prototype methods like `Number.toFixed(123)`